specification/validated-relationships-service-api.yaml

```diff
@@ -602,6 +607,7 @@ paths:
             | 400         | `MISSING_VALUE`                  | Missing header or parameter. For details, see the `diagnostics` field.                                                                    |
             | 400         | `INVALID_VALUE`                  | Invalid header. For details, see the `diagnostics` field.                                                                                 |
             | 400         | `MISSING_IDENTIFIER_VALUE`       | Missing performer:identifier or patient:identifier value.                                                                                 |
+            | 400         | `NOT_SUPPORTED`                  | The request is not currently supported.                                                                                                   |
```

Comment:
Can NOT_SUPPORTED be removed, I don't think the error code is used for this endpoint